test/release/examples/primers/associative.chpl

```diff
@@ -195,12 +201,12 @@ prettyPrint(Scores);
 writeln();
 
 //
-// Let's initialize our 'Scores' array.
+// Let's initialize our ``Scores`` array.
 //
 // Traditional array operations like whole-array assignment and indexing
 // are supported for associative arrays.
 //
-Scores = 100; // every value in 'Scores' is now '100'.
+Scores = 100; // every value in ``Scores`` is now '100'.
```

Comment:
This line comment won't get rendered to rst - so need to wrap, but it doesn't matter too much anyway. (I assume it got caught in a `sed`)